TUTORIAL.md

```diff
@@ -606,22 +606,22 @@ The rest of the Pipeline then runs these in parallel — so if you look at **tre
 If you only have the one agent configured with its two executors, this won't save as much time, but you may have multiple agents on different hardware matching the same label expression.
 
 This script is more complex than the previous ones so it bears some examination.
-You start by grabbing an agent, checking out sources, and making a copy of them using the `archive` step:
+You start by grabbing an agent, checking out sources, and making a copy of them using the `stash` step:
 
 ```groovy
-archive 'pom.xml, src/'
+stash name: 'sources', includes: 'pom.xml,src/'
 ```
 
 is shorthand for the more general:
 
 ```groovy
-step([$class: 'ArtifactArchiver', artifacts: 'pom.xml, src/'])
+step([$class: 'StashStep', includes: 'pom.xml, src/'])
```

Comment:
No actually it is not shorthand for anything; this is obsolete, left over from when the sample actually did use `archive`.